vinaigrette.py

```diff
@@ -0,0 +1,51 @@
+import random
+import time
+import datetime
+
+
+def random_date(start, end, time_format, random_number):
+    """
+    The function parses the strings start and end to struct_time
+    then, converts them to float, multiplies by random
+    and converts it to local time.
+    :param start:
+    :param end:
+    :param time_format: time format of start and end. for example
+    %Y-%m-%d, %m/%d/%Y %I:%M %p etc
+    :param random_number: random number to adding to start
+    :return: date representing by '2022-3-16'
+    """
+    stime = time.mktime(time.strptime(start, time_format))
```

Comment:
Dont use abbreviations in variable names etime can be epoch_time, end_time, estimate_time - you want the next developer that reads your code to have fun reading it, not to guess

Reply:
change this everywhere